### ISSUE TITLE
Search box seems broken.

### ISSUE THREAD

**Issue by fosterrath-mila** [mila-iqia/mila-docs#167]
The search box (implemented within the browser space) seems to spin endlessly. Doesn't provide results even after a long while. 

Is it broken ? 

This is on firefox / linux.

**Comment by fosterrath-mila** [comment]
While looking into this, maybe we could consider adding more value to the search function. There are extensions available to this end such as : 

https://readthedocs-sphinx-search.readthedocs.io/en/latest/

**Comment by fosterrath-mila** [comment]
Might be related to the theme needing an update. See : https://github.com/sphinx-doc/sphinx/issues/8623#issuecomment-752663498

**Comment by btravouillon** [comment]
[Last build](https://github.com/mila-iqia/mila-docs/actions/runs/4492917360/jobs/7903444797):
```
Collecting sphinx-rtd-theme>=0.5.2
    Downloading sphinx_rtd_theme-1.2.0-py2.py3-none-any.whl (2.8 MB)
```

This is the last release available: https://github.com/readthedocs/sphinx_rtd_theme/releases/tag/1.2.0

**Comment by btravouillon** [comment]
Several paths to solve the issue:

1. rebuild `master` -> https://github.com/mila-iqia/mila-docs/actions/runs/4492917360
2. rollback to a release older than sphinx-rtd-theme 1.2.0 -> #170
3. https://github.com/readthedocs/sphinx_rtd_theme/issues/1434 seems to be a good candidate

**Comment by btravouillon** [comment]
Reopen to find a long term solution to this issue.

**Comment by fosterrath-mila** [comment]
I fear this issue will stay opened for a long time. How is the current fix not a long term solution ?

**Comment by btravouillon** [comment]
The current fix pins a dependancy to an older version (`sphinx-rtd-theme < 1.2.0`). I see this as a workaround, not a fix. We can expect it to break one day.

**Comment by btravouillon** [comment]
Initial fix has been reverted in #185, the search box still works now. It seems like the latest `sphinx-rtd-theme>1.2.0` fix the issue.
